src/multiplex/mod.rs

```diff
@@ -23,6 +23,9 @@ pub use self::client::{Client, TagStore};
 pub mod server;
 pub use self::server::Server;
 
+mod pending;
+pub use pending::Pending;
```

Comment:
This will be used as `use tokio_tower::multiplex::Pending`. Do we want to move it to the client module so that it is imported through `use tokio_tower::multiplex::client::Pending` (as before) since this is client-specific?